### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Multi-input autocomplete list]The inline autocomplete is difficult to use because, for example, if "tom" (for “Tomato”) is entered, the first result is "Tomato" and the entry is replaced by this. If now "mato" (for “Tomato”) is entered further, then the value in the field is "Tato", which is an invalid entry. Especially with the screen reader, this will lead to many incorrect entries.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10811]
#### Is this a bug, enhancement, or feature request?
Bug (functional and accessibility) 
#### Describe your proposal.
The inline autocomplete is difficult to use because, for example, if "har" (for “hardware”) is entered, the first result is "IT Hardware (F0200000)" and the entry is replaced by this. If now "dware" (for “hardware”) is entered further, then the value in the field is "IT dware", which is an invalid entry. Especially with the screen reader, this will lead to many incorrect entries. à DD2  Please remove inline autocomplete or change it (only words beginning with the entered character should be displayed)

#### Can you handle that on the application side
No
#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
replicable on https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes
1) make sure the flat list is closed
2) using keyboard , start typing "mato"
3) observe that "M" is converted to "T" in input field and wrong text is being read to user which user didnt type
This will create lot of issues for blind users.

#### Please provide relevant source code (if applicable).
https://ng-15-downport--fundamental-ngx.netlify.app/#/core/multi-input#includes

#### Is there anything else we should know?
Please remove inline autocomplete or change it (only words beginning with the entered character should be displayed)
the input control inside fdp-multi-input has a keyup event on removin that it works fine

**Comment by droshev** [comment]
What we need to modify is if we type a match only by "contains" and not "start with" we should not the auto complete. Example scenario above.

**Comment by KaptaanBarbosa** [comment]
there could be another case where in case value is not there in existing list we might want to call remote api and update the list.

Please reopen this ticket and check we don't need to delete/replace entered text by user if value is not there in existing list we can get it from DB also via API call.

**Comment by I543348** [comment]
We need to try out [autoComplete]="false" as suggested by @mikerodonnell89  , we shall get back after we try this out in case of any further queries.
Thanks.
